docs/design/plugins.md

```diff
@@ -126,7 +126,10 @@ spec:
       metadata:
         name: a
       spec:
-        foo: bar
+        clusterServiceClassExternalName: user-provided-service
+        clusterServicePlanExternalName: default
+        parameters:
+          foo: bar
```

Comment:
Not releated to this PR, but this should look like

```
parameters:
  credentials:
    foo: bar
```

For user-provided-service.